asm/ovl0/xprintf.s

```diff
@@ -3,6 +3,16 @@
 .set gp=64
 .include "macros.inc"
 
+.section .data
+
+glabel D_8003FC00
```

Comment:
does `.balign 4` work here instead of padding bytes?

Reply:
Yep, pushed that change.